### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by joshwallerr** [joshwallerr/newsnotfound#16] [Q&A]
Ask anything and receive an answer!

**Comment by midori404** [comment]
Hi there! Does AI manage the whole news production? Is there any human behind news selection and proofreading before publishing? Thank you!

**Reply by joshwallerr** [reply]
Hey! All articles are created with AI only - there is no human proofreader/middle-man. There are ways to automate proofreading though, which is a feature I am working on in the background :)

**Reply by midori404** [reply]
So the AI selected the topic as well? Cool! I'm just wondering because I read that most articles are related to US, UK, and India. Thanks again!

**Reply by joshwallerr** [reply]
No, the topics are predefined in the code - UK, US and India are just the 3 countries I've added support for right now, but the aim is to slowly build up to including all countries.

The articles are created on a schedule, so each day at specific times, an article for one of the topics will be created. You can see the CRON schedule here - https://github.com/joshwallerr/newsnotfound#cron-schedule
